pyproject.toml

```
@@ -0,0 +1,26 @@
+[tool.black]
+line-length = 88
+target_version = ['py36']
```

Based on the fact that this is being run in a Python 3.8 container
```suggestion
target_version = ['py38']
```